services/common/src/redux/reducers/explosivesPermitReducer.ts

```diff
@@ -14,9 +14,27 @@ const initialState: IExplosivesPermitReducerState = {
 export const explosivesPermitReducer = (state = initialState, action) => {
   switch (action.type) {
     case actionTypes.STORE_EXPLOSIVES_PERMITS:
+      const records = action.payload?.records ?? [];
+      const explosivesPermits = records.map((permit) => {
+        const amendment_count = permit.explosives_permit_amendments.length;
+        const amendments = permit.explosives_permit_amendments.map((a, index) => {
+          const documents = a.documents.map((d) => ({
+            ...d,
+            explosives_permit_document_type_code: d.explosives_permit_amendment_document_type_code,
+          }));
+          return { ...a, documents, amendment_count, isAmendment: true, amendment_no: index + 1 };
```

Comment:
There probably is a change to how things work now but I think for the original permit we want the amendment_no to be zero, would having the index + 1 here cause the original permit to have amendment_no as one?

Reply:
o wait, this is just for the amendments and doesn't include the original I think? if so my mistake please ignore the comment above

Reply:
You got it. 👍 
Original permit gets assigned 0 below.
In some places we have the amendments including the original permit and some places we don't, which can get kind of confusing! 😅